build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        springBootVersion = '2.0.6.RELEASE'
+        springBootVersion = '2.1.0.RELEASE'
```

Comment:
Any reason why we shouldn't be upgrading to a more recent version? I believe 2.2.6 is the latest. @leandrocostawolox any thoughts on this?

Reply:
Totally! Let's use 2.2.6
2.1.0 it's from 2018